<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metro Textures Editor </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QListWidget" name="lstTextures">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="pnlImageView" native="true"/>
          <widget class="QWidget" name="pnlPropertyGrid" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>450</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_textures_bin"/>
   <addaction name="separator"/>
   <addaction name="actionSave_textures_bin"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_texture"/>
   <addaction name="actionRemove_texture"/>
   <addaction name="separator"/>
   <addaction name="actionShow_transparency"/>
   <addaction name="actionCalculate_texture_average_colour"/>
  </widget>
  <action name="actionOpen_textures_bin">
   <property name="icon">
    <iconset resource="../../MetroEX/res/resources.qrc">
     <normaloff>:/imgs/FolderOpened_32x.png</normaloff>:/imgs/FolderOpened_32x.png</iconset>
   </property>
   <property name="text">
    <string>Open textures.bin</string>
   </property>
   <property name="toolTip">
    <string>Open textures.bin</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_textures_bin">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/imgs/Save_32x.png</normaloff>:/imgs/Save_32x.png</iconset>
   </property>
   <property name="text">
    <string>Save textures.bin</string>
   </property>
   <property name="toolTip">
    <string>Save textures.bin</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAdd_texture">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/imgs/InsertPicture_32x.png</normaloff>:/imgs/InsertPicture_32x.png</iconset>
   </property>
   <property name="text">
    <string>Add texture</string>
   </property>
   <property name="toolTip">
    <string>Add texture</string>
   </property>
  </action>
  <action name="actionRemove_texture">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/imgs/SketchflowRemoveScreen_32x.png</normaloff>:/imgs/SketchflowRemoveScreen_32x.png</iconset>
   </property>
   <property name="text">
    <string>Remove texture</string>
   </property>
   <property name="toolTip">
    <string>Remove texture</string>
   </property>
  </action>
  <action name="actionShow_transparency">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../MetroEX/res/resources.qrc">
     <normaloff>:/imgs/Transparency_32x.png</normaloff>:/imgs/Transparency_32x.png</iconset>
   </property>
   <property name="text">
    <string>Show image transparency</string>
   </property>
   <property name="toolTip">
    <string>Show image transparency</string>
   </property>
  </action>
  <action name="actionCalculate_texture_average_colour">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/imgs/ColorWheel_32x.png</normaloff>:/imgs/ColorWheel_32x.png</iconset>
   </property>
   <property name="text">
    <string>Calculate texture average colour</string>
   </property>
   <property name="toolTip">
    <string>Calculate texture average colour</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../MetroEX/res/resources.qrc"/>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
